### PR TITLE
Initialize repo with readme

### PR DESCRIPTION
**Pull request by MTCam**
Do we need to talk about this?  Or just do it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+This repository houses drivers, configurations, and
+experiments that use CEESD codes, including mirgecom.
+
+This repo has the potential to become a large
+parking lot for useless things falling into
+disrepair.  How do we stop it?
+
+ * Periodic CI?
+ * Nightly tests (under what conditions?)
+ * Manifest?  a top-level file of repo contents
```
Comment (inducer) [inline README.md]: Yes, this README.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
 # experiments
```
Comment (inducer) [inline README.md]: Titles use titlecase?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
 Simulation drivers archive
+
+This repository houses drivers, configurations, and
+experiments that use CEESD codes, including mirgecom.
+
+This repo has the potential to become a large
+parking lot for useless things falling into
+disrepair.  How do we stop it?
```
Comment (inducer) [inline README.md]: This reads more like an issue than a README

**Comment by MTCam** [comment]
Ok fixed the readme to something more aligned with the intended real readme.  If we can just commit this, we can continue the discussion on how to structure it in another PR.

**Review by inducer — APPROVED** [review]

**Pull request merged**